src/ConductorSharp.Engine/Builders/DynamicTaskBuilder.cs

```diff
@@ -0,0 +1,56 @@
+﻿using ConductorSharp.Client.Model.Common;
+using ConductorSharp.Engine.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Text;
+
+namespace ConductorSharp.Engine.Builders
+{
+    public class DynamicTaskBuilder<I, O> : BaseTaskBuilder<DynamicTaskInput<I, O>, O>
+    {
+        public const string TaskType = "DYNAMIC";
```

Comment:
These can be private if they are only used in this class

Reply:
For the moment they are only used in this class, so I will make them private. In the future we might have a reason to access this outside of the builder.